build.sh

```diff
@@ -396,7 +400,7 @@ isMSBuildOnNETCoreSupported()
         if [ "$__HostOS" == "Linux" ]; then
             __isMSBuildOnNETCoreSupported=1
             # note: the RIDs below can use globbing patterns
-            UNSUPPORTED_RIDS=("debian.9-x64" "ubuntu.17.04-x64" "alpine.3.6.*-x64")
+            UNSUPPORTED_RIDS=("debian.9-x64" "ubuntu.17.04-x64" "rhel.6-x64")
```

Comment:
Is adding `rhel.6-x64` here intentional?

Reply:
Oops, it is not. It has somehow leaked there from my rhel.6 work.

Reply:
Fixed and amended.